src/search/operator.py

```diff
@@ -63,6 +63,13 @@ class SearchOperator(AsyncModalOperatorMixin, bpy.types.Operator):  # noqa: WPS2
         options={'SKIP_SAVE'},
     )
 
+    next_index: IntProperty(  # type: ignore
+        name='Next index',
+        description='Next index to fill with search results',
+        default=0,
+        options={'SKIP_SAVE'},
```

Comment:
Isso faz com que essa propriedade não seja salva? Talvez me ajude no outro bug se for o caso

Reply:
Acho que isso é pro usuário não poder modificar se tiver um `layout.prop()` na interface, mas não tenho certeza